### ISSUE TITLE
Forum main page broken

### ISSUE THREAD

**Issue by Gatsby-PF** [moxi9/phpfox#1218]
Forum page is showing redundant show buttons.
4.3.0b2 with default bootstrap theme.

![forum](https://cloud.githubusercontent.com/assets/14284710/13794777/8e5dc342-eaf5-11e5-9ee2-88cc36cdff78.png)

**Comment by neil96** [comment]
https://docs.phpfox.com/display/FOX4MAN/Upgrading+Your+phpFox+Website => Might you forgot rebuild bootstrap core.

**Comment by Gatsby-PF** [comment]
No, I did that.  First reset bootstrap them to default theme, then rebuilt the bootstrap core and cleared server and browser caches. The issue did not change.  I will repeat the procedure tomorrow and get back to this.

**Comment by robert9592** [comment]
Can you try to revert the Bootstrap theme and then rebuilt it?

**Comment by neil96** [comment]
Note: Please backup your change before you revert.

**Comment by Gatsby-PF** [comment]
I made bootstrap theme default, I reverted, rebuilt the bootstrap core, then cleared cache, then cleared browser cache. The issue is still there.

**Comment by Gatsby-PF** [comment]
I made bootstrap the default theme, then reverted, then rebuilt bootstrap core, then cleared cache, the cleared browser cache.  The issue persists.

**Comment by neil96** [comment]
Please submit a ticket with your site information. I will check for you.

**Comment by Gatsby-PF** [comment]
Thats okay, in the meantime I isolated this to my theme. For some reason, even after revert/rebuild/clearing the cache and EVERYTHING in the browser, the browser still displayed residual content of the broken page.  I repeated the same procedure above a third time.  Then I closed out of the browser and deleted everything in the sandbox I run the browsers from. Then I started with a fresh sandboxed instance of the browser without any of the crap from the previous browser session.  When I logged in and visited the forum, it was perfect.  So my theme has a css gaff somewhere that I need to isolate.  So it appears to be the browser that was at fault that for some reason refused to display the correct css even after rebuilding the bootstrap core. It is not the script so I am closing this issue.
